x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/helpers.tsx

```diff
@@ -75,20 +75,15 @@ export const getEcsCompliantTableColumns = (): Array<
   {
     field: 'description',
     name: i18n.ECS_DESCRIPTION,
-    render: (description: string | undefined) =>
-      description != null ? (
```

Comment:
For reviewers:

we remove this check because it checks an impossible state.`getEcsCompliantTableColumns` only accepts ecs based fields which by new type definition always have `description` field.